### PR TITLE
logstash 1.5.3

### PR DESCRIPTION
**Pull request by nijikon**

### CODE REVIEW

File: Library/Formula/logstash.rb [hunk 1]
```diff
@@ -20,6 +22,6 @@ def caveats; <<-EOS.undent
 
   test do
     system "#{bin}/logstash", "--version"
-    system "/bin/echo 'hello world' | #{bin}/logstash agent -e 'input { stdin { type => stdin } } output { stdout { codec => rubydebug } }'"
+    system "/bin/echo", "'hello world' | #{bin}/logstash agent -e 'input { stdin { type => stdin } } output { stdout { codec => rubydebug } }'"
```
Comment (dunn) [inline Library/Formula/logstash.rb]: Can this be re-written using `pipe_output`, or would that get too messy?

Comment (nijikon) [inline Library/Formula/logstash.rb]: We can try ;)

Comment (DomT4) [inline Library/Formula/logstash.rb]: It'd be nice to kick it onto two lines instead of the one. It's currently making the formula rather wide.

Comment (mikemcquaid) [inline Library/Formula/logstash.rb]: :+1:

Comment (DomT4) [inline Library/Formula/logstash.rb]: Can we get some movement on this? Seems to be the only thing halting this being merged right now. Thanks!

**Pull request closed without merging**